tests/Integration/ScriptsTest.php

```diff
@@ -382,4 +348,39 @@ public function assert_script_statuses( string $handle, array $assert_true = arr
 			);
 		}
 	}
+
+	/**
+	 * Test that the tracker script is correctly output in HTML markup
+	 * when the wp_parsely_enable_cfasync_attribute filter is used.
+	 *
+	 * @covers \Parsely\Scripts::enqueue_js_tracker
+	 * @uses \Parsely\Parsely::get_asset_cache_buster
+	 * @uses \Parsely\Parsely::api_key_is_missing
+	 * @uses \Parsely\Parsely::api_key_is_set
+	 * @uses \Parsely\Parsely::get_options
+	 * @uses \Parsely\Parsely::post_has_trackable_status
+	 * @uses \Parsely\Parsely::update_metadata_endpoint
+	 * @uses \Parsely\Scripts::register_scripts
+	 * @uses \Parsely\Scripts::script_loader_tag
+	 * @group enqueue-js
+	 * @group insert-js
+	 */
+	public function test_tracker_html_markup_when_cfasync_filter_is_called(): void {
```

Comment:
```suggestion
	public function test_tracker_markup_has_attribute_when_cfasync_filter_is_used(): void {
```

This change will read slightly better with Testdox format.